### PR TITLE
Avoid skipped tests

### PR DESCRIPTION
**Pull request by jordisala1991**
I am targetting this branch, because this only changes tests.

## Subject

Every test is now executed, there are no skipped test. We decrease on test quantity because FixMediaContextCommandTest was extending CommandTest which is a Test class from symfony, not a TestCase.

Not sure about adding a deprecated dependency to required-dev but it was needed to test Amazon. Gauffrete itself uses it on its own Amazon class tho. Don't know if there are any other way of doing it.

### CODE REVIEW

**Comment by greg0ire** [comment]
> We decrease on test quantity because FixMediaContextCommandTest was extending CommandTest which is a Test class from symfony, not a TestCase.

Do you actually mean increase? If we have more actual tests, it should increase, right?

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
         "knplabs/gaufrette": "^0.1.6 || ^0.2",
         "kriswallsmith/buzz": "^0.15",
         "sonata-project/classification-bundle": "^3.0",
-        "sonata-project/core-bundle": "^3.0",
+        "sonata-project/core-bundle": "^3.2",
```
Comment (greg0ire) [inline composer.json]: Why is this change needed?

**Comment by jordisala1991** [comment]
Decrease, we had 250~ test and now 217~. But they cover more code!

File: Tests/Command/FixMediaContextCommandTest.php [hunk 2]
```diff
@@ -17,9 +17,8 @@
 use Sonata\MediaBundle\Provider\Pool;
 use Symfony\Component\Console\Application;
 use Symfony\Component\Console\Tester\CommandTester;
-use Symfony\Component\Console\Tests\Command\CommandTest;
 
-class FixMediaContextCommandTest extends CommandTest
```
Comment (jordisala1991) [inline Tests/Command/FixMediaContextCommandTest.php]: This line was adding extra Symfony own tests @greg0ire

**Comment by greg0ire** [comment]
> CommandTest which is a Test class from symfony, not a TestCase.

 https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Tests/Command/CommandTest.php extends TestCase so yes, it's a test case.

File: composer.json (hunk 1 above)
Comment (jordisala1991) [inline composer.json]: Because on older version we get this when running lower-deps:

Error: Call to protected method PHPUnit_Framework_TestCase::getMockForAbstractClass() from context 'Sonata\CoreBundle\Test\EntityManagerMockFactory'

**Comment by jordisala1991** [comment]
Yes, every test is itself a TestCase, but I mean this class is not ment to be extended, its a test class

File: Tests/Command/FixMediaContextCommandTest.php (hunk 2 above)
Comment (greg0ire) [inline Tests/Command/FixMediaContextCommandTest.php]: I get it now.

**Comment by greg0ire** [comment]
@sonata-project/contributors please review

**Comment by greg0ire** [comment]
Thanks @jordisala1991 !
